pkgs/development/python-modules/hikari/update.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env nix-shell
+#! nix-shell -i bash -p gnused curl jq common-updater-scripts
+# Update version
+VERSION="$(curl -sL "https://api.github.com/repos/buchen/portfolio/tags" | jq '.[0].name' --raw-output)"
+update-source-version portfolio "$VERSION"
+
+# Update Git hash (see ./README.md)
+GIT_HASH=$(curl "https://api.github.com/repos/hikari-py/hikari/tags" | jq 'map(select(.name == "'$VERSION'")) | .[0].commit.sha' | sed 's/"//g')
+sed -i "s/git_hash = \"[a-f0-9]\{40\}\"/git_hash = \"$GIT_HASH\"/g" ./default.nix
```

Comment:
```suggestion
#!/usr/bin/env nix-shell
#! nix-shell -i bash -p gnused curl jq common-updater-scripts
# Update version
TAGS=$(curl -sL "https://api.github.com/repos/hikari-py/hikari/tags")
VERSION="$(echo $TAGS | jq '.[0].name' --raw-output)"
GIT_HASH=$(echo $TAGS | jq 'map(select(.name == "'$VERSION'")) | .[0].commit.sha' --raw-output)

update-source-version "python3Packages.hikari" "$VERSION" --rev=$GIT_HASH
```

Reply:
I found another neat solution to get the commit hash without breaking reproducibility on the nix layer. The pattern used on fastly is to fetch the full git, store the hash and then remove the git folder.

https://github.com/NixOS/nixpkgs/blob/dec7d29544716eadebcc0959c846651fd3b09781/pkgs/misc/fastly/default.nix#L25

Reply:
I have borrowed the code from the Fastly package and adjusted it for this one; everything works as expected, without an update script being involved.